### PR TITLE
chore(CSS): touch up a bunch of style stuff

### LINKED ISSUE #544: Chat - Improve UI
**Task:** In chat/group chats we want to improve the styling of the UI a little 

1. The Name being above the users message looks very confusing, we should have it appear like 
"Hogan - 10 Seconds Ago" all underneath the message 

2. We need to add some spacing in between users when in a group chat 

Screenshot 2023-03-28 at 3 55 41 PM

### PR DESCRIPTION
**Pull request by InfamousVague**

### CODE REVIEW

**Comment by InfamousVague** [comment]
image
image

**Comment by phillsatellite** [comment]
This is looking wayyy better now Matt, I was curious its intended to have the New Chat button smaller than it is on dev? Otherwise all good for me

Branch

Screenshot 2023-03-29 at 10 34 40 AM

Dev

Screenshot 2023-03-29 at 10 35 10 AM

**Review by phillsatellite — APPROVED** [review]

File: ui/src/components/chat/create_group.rs [hunk 1]
```diff
@@ -115,30 +117,19 @@ pub fn create_group(cx: Scope<Props>) -> Element {
...
-                class: "button-container",
-                Button {
-                    text: "Create DM".into(),
-                    appearance: Appearance::Primary,
-                    onpress: move |_| {
-                        log::info!("create dm button");
-                        ch.send(());
-                    },
+            Button {
+                text: "Create DM".into(),
```
Comment (lgmarchi) [inline ui/src/components/chat/create_group.rs:126]: Just remember to use get_local_text here.

**Review by lgmarchi — APPROVED** [review]
Awesome fixes in the UI, much better. 

I would say to not let chat button, search input, create DM button and checked boxes touching scroll bar. 
image

**Pull request merged**